client/src/components/profile/components/TimeLine.js

```diff
@@ -76,11 +84,69 @@ class TimelineInner extends Component {
     this.prevPage = this.prevPage.bind(this);
     this.nextPage = this.nextPage.bind(this);
     this.lastPage = this.lastPage.bind(this);
+    this.renderViewButton = this.renderViewButton.bind(this);
+  }
+
+  renderViewButton(Id, files, githubLink, solution) {
```

Comment:
I'm nit-picking here, could we follow the naming convention and change `Id` to `id`? :slightly_smiling_face:

Reply:
Sure thing.  I will change Id to id and push another commit. 

What should I do about the tests ? . Should I do a separate PR for the tests of TimeLine and CertificateSettings  after this gets merged or just include the tests for TimeLine along with the change to Id to id in a single commit in this PR itself ?

Thanks in advance

Reply:
You can expand the tests for bot timeline and certification settings in here. We have traditionally been lacking in tests, the more the merrier. Let us know if you are stuck.

Reply:
Ok, thanks